experimental/framework/fluid-static/src/containerCode.ts

```diff
@@ -35,7 +35,13 @@ export interface IFluidContainerEvents extends IEvent {
  * FluidContainer defines the interface that the developer will use to interact with Fluid.
  */
 export interface FluidContainer
-    extends Pick<Container, "audience" | "clientId">, IEventProvider<IFluidContainerEvents> {
+    extends Pick<Container,
```

Comment:
I don't think we should be extending Container here -- we want to be thoughtful about reshaping the API surface in places where it's not a great fit, so I'd rather see us explicitly define the API surface here even when it's similar to existing surface on Container.